components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -51,12 +51,12 @@ void MAMMicrophysics::set_defaults_() {
   config_.amicphys.do_coag = true;
 
   config_.amicphys.nucleation = {};
-  config_.amicphys.nucleation.dens_so4a_host = 1770.0;
-  config_.amicphys.nucleation.mw_so4a_host = 115.0;
-  config_.amicphys.nucleation.newnuc_method_user_choice = 2;
-  config_.amicphys.nucleation.pbl_nuc_wang2008_user_choice = 1;
-  config_.amicphys.nucleation.adjust_factor_pbl_ratenucl = 1.0;
-  config_.amicphys.nucleation.accom_coef_h2so4 = 1.0;
+  //config_.amicphys.nucleation.dens_so4a_host = 1770.0;
```

Comment:
These mods are due to the mismatch in the mam4xx submodule. Once microphysics PR is merged, we will rebase to remove these mods.